deployments/cloudfoundry/README.md

```diff
@@ -8,4 +12,14 @@ In this configuration, the Collector will run in the same container as the app.
 ### Bosh Release
 
 This is a Bosh release of the Splunk Collector. This deploys the Collector to the PCF
-environment as a standalone deployment.
\ No newline at end of file
+environment as a standalone deployment.
+
+### Tanzu Tile
+
+This is a Tanzu Tile of the Splunk Collector, which is a packaged release of the collector
```

Comment:
For consistency, should all `Splunk Collector` be changed to `Splunk OpenTelemetry Collector`, e.g.
```suggestion
This is a Tanzu Tile of the Splunk OpenTelemetry Collector, which is a packaged release of the collector
```

Reply:
Good question, I'm not sure if `Splunk Collector` is clear, or if we always need it to be `Splunk OpenTelemetry Collector`. I'll check with the docs team. I do see other references to it without `OpenTelemetry` in this repo, but it's worth checking to see.